apps/react/src/features/genres/components/NavigationList/NavigationList.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC, memo } from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import InboxIcon from '@mui/icons-material/MoveToInbox';
+import MailIcon from '@mui/icons-material/Mail';
+
+import { NavigationProps } from '../../utils/navigationProps';
+
+type NavigationListProps = {
```

Comment:
```suggestion
type Props = {
```
If you are not exporting a type, you can use the usual form of the name